### PR TITLE
acceptance tests with a lot of users (to test paging)

### PR DESCRIPTION
**Pull request by individual-it**
tests that check basic user interaction when the paging is set to a lower number than the amount of users

### CODE REVIEW

**Comment by patrickjahns** [comment]
Do we really need 5000 users? Paging should be triggered in 500er steps?

**Comment by individual-it** [comment]
The current tests do create  2000 users. The 5000 is just a comment for my self about what I planed to do as functionality.
I've just picked a number with a lot of zeros that does not run too long on my computer.

@butonic @tomneedham what is a reasonable number of users for paging tests?

File: tests/acceptance/features/bootstrap/UserLdapContext.php [hunk 1]
```diff
@@ -169,6 +176,70 @@ public function addValueToLdapAttributeOfTheEntry($value, $attribute, $entry) {
...
+	 * @param string $ou
+	 * 
+	 * @return void
+	 */
+	public function createLDAPUsers($amount, $prefix, $ou) {
+		$uidNumberSearch = $this->ldap->searchEntries(
+			'objectClass=posixAccount', null, 0, ['uidNumber']
+		);
+		$maxUidNumber = 0;
+		foreach ($uidNumberSearch as $seachResult) {
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/UserLdapContext.php]: rename seachResult searchResult

File: tests/acceptance/features/paging.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
+@insulated @disablePreviews
+Feature: paging
+
+As an admin
+I want syncing to work correctly dispite the paging settng
```
Comment (phil-davis) [inline tests/acceptance/features/paging.feature]: s/dispite/despite

File: tests/acceptance/features/paging.feature [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	Scenario: login to a system with a lot of users
+		Then it should be possible to login with the username "my-user-000" and password "my-user-000" into the WebUI
+		And I logout
+		Then it should be possible to login with the username "my-user-100" and password "my-user-100" into the WebUI
+		And I logout
+		Then it should be possible to login with the username "my-user-199" and password "my-user-199" into the WebUI
+
+	Scenario: change password on a system with a lot of users
+		And these users are initialized:
+		|username     |password     |
```
Comment (phil-davis) [inline tests/acceptance/features/paging.feature]: lineup columns

**Comment by phil-davis** [comment]
This now needs user_ldap master to be refactored first with the latest core steps, so that CI can pass.

**Pull request merged**